S/SDL2_net/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SDL2_net"
+version = v"2.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/libsdl-org/SDL_net/archive/refs/tags/release-$(version).tar.gz", "1fefe563ea333a2655c32169d03a376a334cdbe39da51fd424bf5f430dec83f4")
```

Comment:
I suggest using a release tarball instead of struggling with bootstrap
```suggestion
    ArchiveSource("https://libsdl.org/projects/SDL_net/release/SDL2_net-$(version).tar.gz",
                  "15ce8a7e5a23dafe8177c8df6e6c79b6749a03fff1e8196742d3571657609d21"),
```

Reply:
thanks for the feedback, i'm pretty new to this